cmake_modules/FindREDIS.cmake

```diff
@@ -0,0 +1,49 @@
+################################################################################
+#    HPCC SYSTEMS software Copyright (C) 2015 HPCC Systems.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+################################################################################
+
+# - Try to find the libhiredislibrary
+# Once done this will define
+#
+#  LIBHEDIS_FOUND - system has the libhiredis library
+#  LIBHIREDIS_INCLUDE_DIR - the libhiredis include directory(s)
+#  LIBHIREDIS_LIBRARY - The library needed to use hiredis
+
+IF (NOT LIBREDIS_FOUND)
+  IF (WIN32)
+    SET (libhiredis "libhiredis")
+    SET (libev "libev")
+  ELSE()
+    SET (libhiredis "hiredis")
+    SET (libev "ev")
```

Comment:
You are not using libev as yet, I think. You should probably split into findRedis and findEv...